components/automate-ui/src/app/app.module.ts

```diff
@@ -25,16 +25,19 @@ import {
 } from '@angular/platform-browser/animations';
 
 // Modules
-import { ApiTokenModule } from './pages/api-token/api-token.module';
+import { ApiTokenModule } from './modules/token/token.module';
 import { AppRoutingModule } from './app-routing.module';
 import { ChefComponentsModule } from './components/chef-components.module';
 import { ChefPipesModule } from './pipes/chef-pipes.module';
 import { ComplianceModule } from './pages/+compliance/compliance.module';
 import { ComplianceSharedModule } from './pages/+compliance/shared/shared.module';
 import { IntegrationsModule } from './pages/integrations/integrations.module';
-import { PolicyModule } from './pages/policy/policy.module';
+import { PolicyModule } from './modules/policy/policy.module';
 import { ProjectModule } from './pages/project/project.module';
-import { RoleModule } from './pages/roles/roles.module';
+import { RoleModule } from './modules/roles/roles.module';
+import { ChefLicenseModule } from 'app/modules/license/license.module';
+import { ChefUserModule } from 'app/modules/user/user.module';
+import { ChefTeamModule } from 'app/modules/team/team.module';
```

Comment:
Would prefer **not** to have the `Chef` prefix on these 3 for consistency. How is that consistent, you ask, because there are `ChefComponentsModule` and `ChefPipesModule` ?
True, but those are, I believe, groups of diverse things.
All the ones without `Chef` are specific things, like policy, project, role, ... and teams, license, and user fall in with that.